arbitrum-docs/for-devs/quickstart-solidity-hardhat.md

```diff
@@ -55,13 +55,16 @@ We'll install the rest of our dependencies as we go.
    - <a data-quicklook-from='smart-contract'>Smart contracts</a> are small programs that execute transactions according to predefined rules. Ethereum's nodes host and execute smart contracts.
    - You can use smart contracts to build decentralized apps (dApps) that use Ethereum's network to process transactions and store data.
    - DApps let users carry their data and identity between applications without having to trust centralized service providers.
-   - People who run Ethereum nodes[^3] can receive rewards for processing and validating transactions on behalf of users and dApps.
+   - People who run Ethereum nodes[^3] can earn $ETH for processing and validating transactions on behalf of users and dApps.
```

Comment:
Although we explain this at [^3], maybe we could say `specific Ethereum nodes` here could be better